### PR TITLE
Add panacea chain

### PR DESCRIPTION
**Pull request by gyuguen**

### CODE REVIEW

**Comment by gyuguen** [comment]
I don't know if our name will be **medibloc** or **panacea**.

File: medibloc/chain.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "panacea",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Panacea",
```
Comment (youngjoon-lee) [inline medibloc/chain.json]:
```suggestion
    "pretty_name": "MediBloc",
```
I think the name `MediBloc` is more comprehensive for users who are already familiar with CEXs.

Comment (youngjoon-lee) [reply medibloc/chain.json]: Ah.. Let's go with the `Panacea` as you wrote first because MediBloc might launch multiple chains in the future.

**Comment by gyuguen** [comment]
This PR has been recreated [here](https://github.com/cosmos/chain-registry/pull/49), so we terminate it.

**Pull request closed without merging**